README.md

```diff
@@ -1 +1,73 @@
 # pad-array
+
+[![NPM version][npm-image]][npm-url]
+[![build status][travis-image]][travis-url]
+[![David deps][david-image]][david-url]
+[![npm download][download-image]][download-url]
+
+Function to fill an array in Javascript
+
+This code is based in the MATLAB's code [padarray](http://www.mathworks.com/help/images/ref/padarray.html)
+
+## Installation
+
+`npm install ml-pad-array`
+
+## pad-array(data, [options])
+
+Pads the array `data` given the current options`.
```

Comment:
may be good to specify that it doesn't change the input array and returns a new one